pkg/trace/stats/aggregation.go

```diff
@@ -64,8 +80,35 @@ func getStatusCode(s *pb.Span) uint32 {
 }
 
 // NewAggregationFromSpan creates a new aggregation from the provided span and env
-func NewAggregationFromSpan(s *pb.Span, origin string, aggKey PayloadAggregationKey, enablePeerSvcAgg bool) Aggregation {
+func NewAggregationFromSpan(s *pb.Span, origin string, aggKey PayloadAggregationKey, enablePeerSvcAgg bool, customTagConf []string, customTagSpanNames []string) Aggregation {
 	synthetics := strings.HasPrefix(origin, tagSynthetics)
+
+	i := 0
+
+	customKey := CustomTagKey("")
+
+	if len(customTagSpanNames) == 0 {
+		customTags := make([]string, len(customTagConf))
+		for k := range customTagConf {
```

Comment:
What if we sort the tags in `customTagConf`? That way we can guarantee a consistent insertion order when we fetch the values and make the resulting `CustomTagKey`.